src/engine/ruleset/integrations/windows/rules/defense-evasion-timestomp-sysmon.yml

```diff
@@ -0,0 +1,50 @@
+name: rule/defense-evasion-timestomp-sysmon/0
+
+metadata:
```

Comment:
Isn't it necessary to add the compatibility section?

Reply:
ditto